cfme/tests/physical_infrastructure/api/test_physical_server_inventory_api.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils import testgen
+from cfme.utils.rest import assert_response
+
+pytestmark = [pytest.mark.tier(3)]
+
+pytest_generate_tests = testgen.generate([LenovoProvider], scope="class")
+
+
+@pytest.fixture(scope="class")
+def physical_server(provider, appliance):
+    if not provider.exists:
+        provider.create_rest()
+        provider.refresh_provider_relationships
+        provider.validate()
```

Comment:
**REQUIRED**
Please use `setup_provider_modscope` fixture:
```python
def physical_server(setup_provider_modscope, appliance):
```

Reply:
@quarckster I guess the purpose is to use REST for provider provisioning.
@douglasgabriel 
```python
provider.create_rest(check_existing=True, validate_inventory=True)
```
is equivalent to your code.

Reply:
@mkoura `setup_provider_modscope` setup providers via REST https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/provider.py#L488

Reply:
@quarckster sounds like good idea to me. Lenovo is the only physical provider we have ATM, so if it's possible to set it up using REST without issues, ``physical`` should be added to the list.